en/index.md

```diff
@@ -9,15 +9,16 @@ to the left, and presented here as a single page to aid searching.
 
 ## Diátaxis
 
-Use Diátaxis to structure product documentation. A documentation page should
-fit within one of the following categories:
+Use Diátaxis to structure product documentation.
+Diátaxis provides rules that help you identify the type of document that you are writing.
+Types of document include:
 
 - Tutorial
-- How-to
-- Explanation
+- How-to guides
+- Explanations
```

Comment:
I think it should be "Explanation" (singular)?

Reply:
Sure. I will change this.

Reply:
Done.